CSharp/ArticleProjects/ApplicationInsights/README.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Monitoring and debugging Azure Batch applications with Application Insights - Azure | Microsoft Docs
+description: Add instrumentation to your applications to diagnose and debug application running in Azure Batch in near-real time.
+services: batch
+documentationcenter: ''
+author: tamram
+manager: timlt
+editor: ''
+
+ms.assetid: e14e611d-12cd-4671-91dc-bc506dc853e5
+ms.service: batch
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: multiple
+ms.workload: big-compute
+ms.date: 02/01/2017
+ms.author: tamram
+
+---
+# Monitoring and debugging Azure Batch applications with Application Insights
+
+Application Insights provides an elegant and powerful way to monitor and debug 
+applications deployed to Azure services. Using Application Insights you can 
+monitor performance counters and exceptions as well as instrument your code 
+with custom metrics and tracing. Integrating Application Insights with your 
+Azure Batch application allows developers to gain deep insights into behaviors 
+and investigate issues in near-real-time.
+
+This article shows how to add and configure the Application Insights library 
+into your solution and instrument your application code. Futhermore it provides
+examples on how to monitor your application via the Azure portal and build 
+custom dashboards.
+
+*Should this be added in the learn more section?*
```

Comment:
It seems that this was authored for acom and doesn't render as beautifully here?

i.e. these comments show up

Same below where `!WARNING` shows up

Reply:
Seems to render correctly.

Reply:
This is a TODO though right? Should we remove it?